identicon.js

```diff
@@ -164,8 +164,8 @@
         rectangles: null,
 
         color: function(r, g, b, a){
-            var values = [r, g, b, a ? a/255 : 1].map(Math.round);
-            return 'rgba(' + values.join(',') + ')';
+            var values = [r, g, b].map(Math.round);
```

Comment:
I think we still want the alpha to go through round(). Let's just avoid turning 0 into 1. What about:
```
a !== undefined ? a/255 : 1
```